app/src/main/java/org/mozilla/fenix/wallpapers/WallpapersUseCases.kt

```diff
@@ -14,6 +14,7 @@ import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 import mozilla.components.concept.fetch.Client
 import mozilla.components.support.locale.LocaleManager
+import org.mozilla.fenix.FeatureFlags
```

Comment:
nit: would it be possible to have the companion object at the bottom of this file/class? It's a little hard to read currently with the use cause API vals at the top and the companion object in the middle of all of the actual use case code.

Reply:
The companion object for the `LegacyInitializeUseCase`? It would mean shifting that whole class to the bottom. I would be okay with that, but I would argue against moving the new initialize use case down as well, to help preserve the implied suggestion that `Initialize` should be called first.

Reply:
I could also move the companion to the top-level `WallpaperUseCase`  to achieve that

Reply:
Oh my bad, I didn't see it was just in `LegacyInitializeUseCase`. Curly braces are weird, and it looked like the companion object was a part of the top-level class.

Reply:
Yeah, so the whole thing should be deleted when we delete the Legacy case as well. Part of the intent with all these changes is to remove all that local metadata in the first place